src/frontend/src/pages/FinancePage/FinanceComponents/ReportRefundModal.tsx

```diff
@@ -41,15 +45,16 @@ const ReportRefundModal: React.FC<ReportRefundProps> = ({ modalShow, handleClose
   } = useForm({
     resolver: yupResolver(schema),
     defaultValues: {
-      refundAmount: ''
+      refundAmount: '',
+      dateReceived: new Date()
     },
     mode: 'onChange'
   });
 
-  const handleConfirm = async (data: { refundAmount: number }) => {
+  const handleConfirm = async (data: { refundAmount: number; dateReceived: Date }) => {
     handleClose();
     try {
-      await mutateAsync({ refundAmount: data.refundAmount * 100 });
+      await mutateAsync({ refundAmount: data.refundAmount * 100, dateReceived: data.dateReceived.toISOString() });
```

Comment:
can you wrap the data.refundAmount * 100 in a Math.round, I got a floating point error when I tried to submit 2.30 for a refund